students/jayjohnson/lesson02/notes.py

```diff
@@ -0,0 +1,96 @@
+'''
+Jay Johnson lesson 02 notes
+
+'''
+
+# functions make programs
+def addOne(x):
+    result = x + 1
+    return result
+
+# key word arguments or qwards
+# positional arguments
+# you can leave off the return statement
+# you can return multiple values
+#def add():
+#   return 1, 2, "blah"
+# will return it as a tuple
+# local variable = variables in scope
+# pep8???
```

Comment:
Not sure if you wanted me to read this, but you put it in your PR, so yeah, PEP 8 is the style python uses for the standard library and what we're trying to teach in this class. When you have a minute check this out: https://www.python.org/dev/peps/pep-0008/ 

And when you get bored of trying to read it, just install a linter that will yell at you when you're not conforming to the style.